_posts/2018-08-31-zenika-craftsman-swap.md

```diff
@@ -0,0 +1,78 @@
+
+In the afternoon we’ve done a kata focused on hexagonal architecture. We developed a simple application that reads from an external source, let’s say a file, processes that input and outputs to the console. We’ve created a few abstractions that served as adapters and a single implementation that was the port in this case. Although I was familiar with the concept, it was good to debate about the topic, listen to opinions that reinforced my current understanding, and also some challenging ideas about where the domain logic should live, what we consider presentation logic, etc. 
+
+After the kata, we discussed the different TDD styles, outside-in, inside-out and Mael showed me a workshop that he prepared around <a href="https://gojko.net/books/specification-by-example/">Specification by example</a>
+
+# Day 5 - Zenika
+In the last day I met [Xavier](http://twitter.com/xdetant), who is the CTO of the Paris office. We had a chat about how Zenika internally works and its culture. He showed me how Zenika is structured horizontally, with an almost flat hierarchy, how the teams are self organised and all the members have complementary skills. In terms of internal culture, they have many interesting things including “tribes” (similar to the Spotify Model), which are groups where they share focused technical knowledge, and the idea box where people are empowered to implement their own ideas in the organisation. 
+
+That day I also discovered how much I suck at table football. Those guys are really good — I think I didn’t even see the ball during the matches. Thanks for that. :)
+
+<center><img src="{{ site.baseurl }}/assets/custom/img/blog/2018-08-31-zenika-craftsman-swap/tablesoccer.jpg" alt="table soccer" width="550" height="322" class="img-responsive" /></center>
+<br/>
+After lunch, Xavier organized a mobbing session where we used Haskell to solve the "Potter Kata". This was a surprise to me, I didn't expect to do any Haskell during this visit. We were using TDD in baby steps to solve the exercise and we created a few types to represent some domain concepts. Thanks to that exercise I discovered how to deal with RealFloat and Fractional and a few conversion methods between numbers in Haskell.
+
+
+# Conclusion
+
+It is always good to be outside our comfort zone, meet new people, and see different solutions to similar problems. Some of the highlights of this exchange include: 
+
+Meet like-minded craftspeople
```

Comment:
Was this meant to be formatted as a bulleted list? If so it should start with ` * ` like so: 

 * an entry
 * another entry

Otherwise, the newlines are ignored and it comes out as one long sentence.